packages/fast-usdc/src/cli/transfer.js

```diff
@@ -32,7 +32,7 @@ const transfer = async (
       { chainName: 'agoric', rpcAddrs: [config.agoricRpc] },
     );
     const agoricAddr = await queryFastUSDCLocalChainAccount(vstorage, out);
-    const appendedAddr = `${agoricAddr}+${destination}`;
+    const appendedAddr = `${agoricAddr}?EUD=${destination}`;
```

Comment:
This implementation assumes `destination` doesn't need URL encoding. I suppose that's true for any correct bech32 addresses. But I'm not sure we validate that `destination` is a correct bech32 address.

`packages/fast-usdc/src/utils/address.js` has some `addressTools`. I wonder if a function there to combine these is worthwhile. I'm postponing until the need is more evident.

Reply:
I had to fix this anyway. It will now encode all of `appendedAddr`, which I think should work regardless.